src/app/pages/storage/disks/smart-results/smart-results.component.ts

```diff
@@ -32,6 +32,7 @@ export class SmartResultsComponent {
     preInit(entityForm: any) {
         this.aroute.params.subscribe(params => {
             this.disk = params['pk'];
+            this.title = T(`S.M.A.R.T Test Results of ${this.disk}`);
```

Comment:
this won't work for translations, use translate.get to get the text then append the this.disk to it